vignettes/tidyhydat_hydat_db.Rmd

```diff
@@ -0,0 +1,73 @@
+---
+title: "Stepping into the HYDAT Database"
+author: "Dewey Dunnington"
+date: "`r Sys.Date()`"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{tidyhydat_db}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r setup, include = FALSE}
+library(tidyhydat)
+library(dplyr)
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+The HYDAT database is a massive hydrologic data resource. The functions in this package are designed to get the most out of the HYDAT database as quickly as possible, however in the process of reformatting the data to be more useful, some information is lost. This vignette is intended to demonstrate how to access tables within the database for your own custom HYDAT analysis, should additional information be needed.
```

Comment:
What do you mean by "information lost"? I worry this implies that we are obscuring something from the user which are not.

Reply:
I also wonder if some discussion for users letting them now that usually you won't need anything like this. This is advanced functionality.